homeassistant/components/renson/entity.py

```diff
@@ -0,0 +1,35 @@
+"""Entity class for Renson ventilation unit."""
+from __future__ import annotations
+
+from renson_endura_delta.renson import RensonVentilation
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import RensonCoordinator
+from .const import DOMAIN
+
+
+class RensonEntity(CoordinatorEntity):
+    """Renson entity."""
+
+    def __init__(
+        self, name: str, api: RensonVentilation, coordinator: RensonCoordinator
+    ) -> None:
+        """Initialize the ComfoConnect fan."""
```

Comment:
Docstring should be updated to reference the fact this is a base entity class, not necessarily a fan.